lib/util/miq_apache/miq_apache.rb

```diff
@@ -125,7 +125,7 @@ def self.status(full = true)
     end
 
     def self.version
-       MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' httpd")
+      MiqUtil.runcmd("rpm -qa --queryformat '%{VERSION}' #{MiqApache.package_name}")
```

Comment:
Agreed, I think PRs like this are great examples of things we can extract to LinuxAdmin.